src/app/utils/formValidationUtils.js

```diff
@@ -156,19 +142,19 @@ function validate(form, cb) {
       validate: () => true,
       errorMsg: '',
     },
-    'starting-page': {
-      validate: (val) => isNumeric(val),
-      errorMsg: 'Please enter the starting page',
+    startPage: {
+      validate: (val) => (isNumeric(val) && val > '0'),
+      errorMsg: 'Please enter the start page',
     },
-    'ending-page': {
-      validate: (val) => isNumeric(val),
+    endPage: {
+      validate: (val) => (isNumeric(val) && val > '0'),
```

Comment:
I ran into this article yesterday. It is quite clear and interesting about how js handle this format converting. I think it should be fine doing so.

http://2ality.com/2012/03/converting-to-string.html

But of course we can make the code less risky.